theory/RegexAnswers.js

```diff
@@ -0,0 +1,25 @@
+let regex1 = /(antelope)s? (rock)s? out/g;
+
+console.log(regex1.test("antelope rocks out"));
+//console.log(regex1.test("antelopes rock out"));
+
+let regex2 = /[^b]oat/g;
```

Comment:
If we did `[^b]`, then the words coat and oat will match. Better to use a capture group:

```
(g|m)oat
```